test/stepDefinitions/validationSteps.js

```diff
@@ -145,6 +145,7 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
   });
 
   When(/^I enter '(.*)' into the '(.*)' field$/, async function (value, fieldType) {
+    browser.wait(ExpectedConditions.presenceOf(element(by.css('ccd-write-money-gbp-field'))), 5000);
```

Comment:
This would only work for case types which have a money field on their first page. But since it passes with the  current test data and case-management-web isn't expecting more changes, a more generic solution probably not worth looking into.